RpcWallet/RpcWallet.cs

```diff
@@ -334,6 +347,11 @@ private JObject SendMany(UInt160 from, JArray to, Fixed8 fee, UInt160 change_add
             if (fee < Fixed8.Zero)
                 throw new RpcException(-32602, "Invalid params");
             Transaction tx = Wallet.MakeTransaction(null, outputs, from: from, change_address: change_address, fee: fee);
+            if (tx.Size > 1024)
+            {
+                fee += Fixed8.FromDecimal(tx.Size * 0.00001m);
```

Comment:
If fee is less than `.0005` Before this line probably should add the difference so it will have some headroom in case more gas utxo’s have to be added to accommodate the additional tx size.

Reply:
Just fixed it.